frontend/src/lib/components/ActionSelectTab.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+import { capitalizeFirstLetter } from '~/lib/utils'
+
+export default class ActionSelectTab extends Component {
+    render() {
+        let { entityType, chooseEntityType, allTypes } = this.props
+        return (
+            <div style={{ display: 'flex', flexDirection: 'row', height: '25px', borderBottom: '1px solid #cccccc' }}>
+                {allTypes.map((type, index) => (
+                    <div
+                        key={index}
+                        style={{
+                            backgroundColor: entityType == type ? 'white' : '#eeeeee',
+                            flex: 1,
+                            display: 'flex',
+                            justifyContent: 'center',
+                            borderTopLeftRadius: '5px',
```

Comment:
This needs to be the other way around if it's the second tab 
![image](https://user-images.githubusercontent.com/1727427/78463199-f8a49700-76d1-11ea-90d5-9bb61d11d69c.png)